torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -2061,6 +2067,22 @@ void InitXlaModuleBindings(py::module m) {
           }
           return handles;
         });
+  m.def("_xla_mark_tensor",
+        [](const at::Tensor& input, const std::string& info) {
+          TORCH_LAZY_COUNTER("XlaMarkTensor", 1);
+          at::Tensor result;
+          {
+            NoGilSection nogil;
+            result = MarkTensor(input, info);
+          }
+          return result;
+        });
+
+  m.def("_xla_mark_dynamic", [](const at::Tensor& input, uint32_t dim) {
```

Comment:
Looks like this got copied accidentally?

Reply:
Thanks for catching it! Removed